src/ParallelAlgorithms/DiscontinuousGalerkin/FluxCommunication.hpp

```diff
@@ -292,7 +292,7 @@ struct ReceiveDataForFluxes<
                    const gsl::not_null<db::item_type<
                        Tags::Mortars<receive_temporal_id_tag, volume_dim>>*>
                        neighbor_next_temporal_ids,
-                   const db::const_item_type<receive_temporal_id_tag>&
+                   const typename receive_temporal_id_tag::type&
```

Comment:
The temporal ID might be a compute tag, which this change would break. That's related to an earlier comment and how I'm worried that removing the `const_item_type` logic actually makes the code harder to maintain